reactioncommerce/reaction
# Create migration files for upgrade from 1.0 to 1.5

Resolves in part #2938

**How To Test:**
- On an updated marketplace branch, git checkout to commit `91e18f26b7bc`. That's v1.0.0 commit. (You can also checkout into the tag if you have fetched it earlier).
- After `reaction reset -y && reaction`, create a few orders, and also add a product to cart without checking out.
- After that, checkout back to marketplace. Then run `reaction pull marketplace` (I do this just to use cli to re-instal modules). Then, fetch+checkout into branch `seun-migration-1.0-1.5`.
- Start `reaction` again. Confirm that migrations run up to version 15.
- Visit the Order dashboard, visit the PDP, and other pages. The goal is to have everything run smoothly as it should. This PR at the moment doesn't fully solve that. It gets us to the point of loading the app successfully, and being able to see Order dashboard (with shipping working).

Next up to resolve will be *Refunds* (which is the error you'll be seeing in your console). This PR as is can still be merged will I start working separately to fix the other issues. 

Other issues found (if related to migration) can be reported on the main ticket #2938.
